a2a/1.0/modules/ROOT/pages/a2a-connector-studio.adoc

```diff
@@ -0,0 +1,138 @@
+
+* *Task Listener*
++
+This source is an extension of HTTP Listener. It listens for incoming tasks directed towards its agent, such as messages or API calls.
++
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-steps]
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+include::connectors::partial$t-studio-partial.adoc[tags=add-connector-operation]
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-intro]
+
+include::connectors::partial$t-studio-partial.adoc[tags=property-placeholders]
+
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-first-steps]
+
+. In the *General* tab, in *Connection*, select the authentication method to configure:
+* <<expression-or-bean-ref>>
+* <<basic_authentication>>
+* <<digest>>
+* <<ntlm>>
+
+ifdef::tls[]
+include::connectors::partial$t-studio-partial.adoc[tags=global-element-tls-steps]
+endif::[]
+
+[[expression-or-bean-ref]]
+=== Expression or Bean Reference
+
+Uses a DataWeave expression or bean reference to define authentication.
+
+Enter the authentication information in the *Global Element Properties* screen:
+
+. Select *General > Authentication* and enter or set the values for these fields:
+. In *Authentication*, select *Expression or Bean reference* and enter the DataWeave expression. 
+. Click *OK*. 
+
+[[basic_authentication]]
+=== Basic Authentication
+
+Basic authentication is a simple (less secure) authentication scheme where the client sends the username and password in plain text (encoded in Base64) in the HTTP Authorization header.
+
+Enter the Basic authentication information in the *Global Element Properties* screen:
+
+. Select *General > Authentication* and enter or set the values for these fields:
+. In *Authentication*, select *Basic authentication* and enter or set the values for these fields:
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Username | Username used to initialize the session.
+|Password | Password used to authenticate the user.
+|Preemptive | If set to true, the client sends its authentication credentials (username and password) with the initial HTTP request to the server, rather than waiting for the server to challenge (401 Unauthorized) the client.
+|===
+. Click *OK*.
+
+[[digest]]
+=== Digest
+
+Digest is an HTTP authentication scheme that uses a challenge-response mechanism to avoid sending passwords in plain text. 
+
+Enter the Digest authentication information in the *Global Element Properties* screen:
+
+. Select *General > Authentication* and enter or set the values for these fields:
+. In *Authentication*, select *Digest authentication* and enter or set the values for these fields:
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Username | Username used to initialize the session.
```

Comment:
double checking these are the same fields as basic auth?

Reply:
Yes, they're the same.